com.endlessm.HackSoundServer.json.in

```diff
@@ -6,7 +6,13 @@
     "sdk": "org.gnome.Sdk",
     "finish-args": [
         "--share=ipc",
-        "--socket=pulseaudio"
+        "--socket=pulseaudio",
+        "--talk-name=com.endlessm.dinosaurs.en",
+        "--talk-name=com.endlessm.encyclopedia.en",
+        "--talk-name=com.endlessm.Hackdex_chapter_one",
+        "--talk-name=com.endlessm.Fizzics",
+        "--talk-name=com.endlessm.HackUnlock",
+        "--talk-name=com.endlessm.OperatingSystemApp"
```

Comment:
Whether or not we do the global thing, having to maintain this list here would be a headache.

Reply:
> Yes. Another option (at least I am not missing something in regard to the sandbox permissions) would be to restrict the PlaySoundAtBusName method to receive only "unique bus names" (for example, ":1:123" instead of "com.endlessm.Fizzics").
> 

Not sure I understand, because the server is already receiving that info as `sender` https://github.com/endlessm/hack-sound-server/blob/master/src/server.py#L477

> So we would have to get the "unique bus name" from the target application in hack-toolbox-app.
> 

Yes, and the owner of that sound, in the lockscreen case, is the toolbox, we don't really care about what app is the toolbox connected to.

> I was thinking on using `org.freedesktop.DBus.GetNameOwner` to get the unique name in the hack-toolbox-app, but now I am not sure if (because it runs on a flatpak) it wouldn't be able to find the owner for the target application... and it would involve to just move ^^^ these lines to the hack-toolbox-app.